examples/acoustic/gradient_example.py

```diff
@@ -98,23 +97,20 @@ def run(dimensions=(50, 50, 50), spacing=(15.0, 15.0, 15.0), tn=750.0,
 
     # Forward Operators - one with save = True and one with save = False
     fw = ForwardOperator(model, u_save, src, rec_t, time_order=time_order,
-                         spc_order=space_order, save=True, cache_blocking=cache_blocking,
-                         dse=dse, dle=dle, compiler=compiler, profile=True)
+                         spc_order=space_order, save=True, dse=dse, dle=dle)
 
     fw_nosave = ForwardOperator(model, u_nosave, src, rec_t, time_order=time_order,
-                                spc_order=space_order, save=False,
-                                cache_blocking=cache_blocking, dse=dse, dle=dle,
-                                compiler=compiler, profile=True)
+                                spc_order=space_order, save=False, dse=dse, dle=dle)
 
     # Calculate receiver data for true velocity
-    fw_nosave.apply(autotuning=autotuning, rec_t=rec_t)
```

Comment:
OK, so, I thought about this and eventually I opted for leaving it as is. I think we can safely show autotuning in another example